src/ralph/dns/views.py

```diff
@@ -71,14 +73,26 @@ def post(self, request, *args, **kwargs):
         if posted_form.is_valid():
             if posted_form.data.get('delete'):
                 errors = self.dnsaas.delete_dns_record(form.data['pk'])
+                if not errors:
+                    messages.success(
+                        request, _('DNS record has been deleted.')
+                    )
             elif posted_form.cleaned_data.get('pk'):
                 errors = self.dnsaas.update_dns_record(
                     posted_form.cleaned_data
                 )
+                if not errors:
+                    messages.success(
+                        request, _('DNS record has been saved.')
```

Comment:
if you use deleted/created in other cases, then there `updated` suits better than `saved`.